source/tutorials/setap.rst

```diff
@@ -0,0 +1,373 @@
+be found in the `Format specification document`_ and the
+`Algorithm Theoretical Baseline Document`_ respectively.
+
+A `Python API`_ that allows to easily access the product and perform basic
+operations is also available.
+
+In general all main information and news about the S1-ETAD product are
+collected in the `Product Page`_ on the ESA web site.
+
+.. _`Product Definition Document`: https://sentinels.copernicus.eu/documents/247904/4629150/Sentinel-1-Product-Definition-Document.pdf
+.. _`Algorithm Theoretical Baseline Document`: https://sentinels.copernicus.eu/documents/247904/4629150/Sentinel-1-ETAD-Algorithm-Technical-Baseline-Document.pdf
+.. _`Python API`: https://gitlab.com/s1-etad/s1-etad
+.. _`Product Page`: https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1/data-products/etad-dataset
+
+.. _`sentinel-1-annotations-caching-policy-label`:
+**Sentinel-1 Annotations caching policy**
+
+The "Sentinel-1 Annotations” products (cf. "EO Data" menu) benefit from a
+data caching mechanism on GEP.
+The cache (Manifest and Annotation files) is managed for a pre-defined set
+of geographic areas and time span.
+These areas are specified according to the seismic areas defined by the
+`CEOS Seismic Hazards`_ work group, as follows:
+
+* mask_eu (Europe seismic areas)
+* mask1 (20% of he world's seismic areas)
+* ask2 (35%)
+* mask3 (50%)
+* mask70 (70%)
+* mask100 (100%)
+
+For the launch of the GEP SETAP service (September 2021), the cache is
+initialised as follows:
+
+* mask_eu (Europe)
+* January-September 2021
+* adding up new 2021 acquisitions from September 2021 onwards
+
+Moreover, when a processing Job is started over a spatial and temporal
+data search outside of the current data caching automation, an opportunistic
+approach will keep in cache the retrieved input products.
+
+This is resulting in an additional cache capacity, mutualised, that is
+growing based on user activities (so, without a planification on the GEP
+side, but with a monitoring of the user interests):
+
+* user selected job inputs falling outside current cached products,
+  for spatial and temporal domains with some limitations (see further
+  below the FAQ section, :ref:`Q1<faq-q1-label>_)
+
+For the future of SETAP operations on GEP, a plan with progressive caching
+based on these CEOS tectonic areas can be achieved according to user needs
+gathered through user engagement activities. For example:
+
+* the European seismic area,
+* then extended to cover 20% of the world seismic zones,
+* then 50%
+* and potentially up to 100% of the CEOS-defined seismic areas worldwide
+
+In terms of performances of a processing job, SETAP will be the fastest when
+accessing pre-cached input products, a little bit slower if products not yet
+in cache can be accessed from the Copernicus SciHub, and quite slow for older
+products only available from other remote data repositories
+(e.g. the Alaska Science Facility) connected to GEP.
+
```

Comment:
@HerveCaumont I would also add the following sentence:

Please note that processing jobs including products that are out of cache could fail it the provider is not able to make that products available within the system configured timeout (e.g. this could happen sometimes for product stored in the Long Term Archive on the Copernicus SciHub).

Reply:
We can discard it

Reply:
The change is not necessary